glean_parser/validate_ping.py

```diff
@@ -59,17 +50,18 @@ def _validate_ping(ins, outs, schema_url):
     return has_error
 
 
-def validate_ping(ins, outs=None, schema_url=PING_SCHEMA_DEFAULT_URL):
+def validate_ping(ins, outs=None, schema_url=None):
     """
     Validates the contents of a Glean ping.
 
     :param ins: Input stream or file path to the ping contents to validate
     :param outs: Output stream to write errors to. (Defaults to stdout)
     :param schema_url: HTTP URL or local filesystem path to Glean ping schema.
-        Defaults to the current version of the schema in
-        mozilla-pipeline-schemas.
     :rtype: int 1 if any errors occurred, otherwise 0.
     """
+    if schema_url is None:
+        raise ValueError("Missing required argument 'schema_url'")
```

Comment:
Missing arguments are usually `TypeError` in Python (not sure why, it doesn't make much sense, but let's be consistent with that)...